frontend/packages/core/src/Table/table.tsx

```diff
@@ -132,64 +136,84 @@ interface TableProps extends Pick<MuiTableProps, "stickyHeader"> {
   children?:
     | (React.ReactElement<TableRowProps> | null | undefined | {})[]
     | React.ReactElement<TableRowProps>;
+  order?: Order;
+  orderBy?: string;
```

Comment:
Im confused why we need these but maybe I'm missing something.

I would think we would have a column prop of 'sortable' or something. If you click on it then it would call the `onRequestSort` if available and then internally keep track of the order/orderBy, if `onRequestSort` isn't available then we could just sort it internally as if it has all the data.

We could have a `defaultSort` param if necessary but I don't think the caller should need this data externally.

Reply:
I based the implementation on the example from the MUI docs, basically only one column is active at a time, that's the orderBy prop, I'll add a video in the description of the PR so it's more clear